test/tests/php-hello-world/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+IMAGE="$1"
+DIR_TEST="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+DIR_CONTAINER="/var/www/html/hello-world"
+
+docker run -it --rm -v $DIR_TEST:$DIR_CONTAINER -w $DIR_CONTAINER $IMAGE php index.php
```

Comment:
Can you please adapt the style of this file to match other examples in the repo? (ie, lowercase/camelCase local variables, quoted variables in the `docker run` line, prefer single quotes where possible like `DIR_CONTAINER`'s value)

Also, can you add an `expected-std-out.txt` file like we have in [`hylang-sh`](https://github.com/docker-library/official-images/tree/master/test/tests/hylang-sh) for example so that we verify that the output is exactly what we expect as well?